### ISSUE TITLE
Validate man pages for rootless config location

### ISSUE THREAD

**Issue by TomSweeneyRedHat** [containers/libpod#3407]
I stole the location for the configuration files from some man pages saying they files were located in XDG_RUNTIME_DIR.  However as pointed out in #3404 they really live in ${HOME}

Adding this as a reminder for myself to find and scrub.

**Comment by DisposaBoy** [comment]
PR https://github.com/containers/libpod/pull/3404 is incorrect. You just got the name of the env var wrong. The correct name is *XDG_**CONFIG_HOME***.

See https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html for docs.

**Comment by rhatdan** [comment]
Could you open a PR to change the code as well as the help message to do the "RIGHT" thing.

**Comment by TomSweeneyRedHat** [comment]
@DisposaBoy I'm not entirely clear on your last line:
"EDIT: my mistake, the code does in fact hard-code this path — which is the wrong thing to do."

Which path are you seeing hardcoded?  I think you're looking at lines like this next one, but just wanted to be sure we were on the same page.

```
./cmd/podman/remoteclientconfig/config_darwin.go:	return filepath.Join(homeDir, ".config", "containers", remoteConfigFileName)
```

**Comment by DisposaBoy** [comment]
I can try and make a PR later this week/next week if I have time, but yes... I'm talking about the hard-coded paths like `join(home, .config)`. Those are the default directories, so things will work most of the time, but it doesn't actually follow the XDG spec. Those code paths should be using the `XDG_CONFIG_HOME` env var and only use the hard-coded paths if it's not set.

You can probably just re-use one of the existing package for this like https://github.com/containers/libpod/blob/master/vendor/github.com/docker/docker/pkg/homedir/homedir_linux.go#L100

**Comment by TomSweeneyRedHat** [comment]
@DisposaBoy if you've time, would love a review on #3466.

**Comment by TomSweeneyRedHat** [comment]
Fixed with #3466
